packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -152,36 +152,20 @@ runTestCheckAgainstSpec(
   returnTypes,
   testDatas,
   {
-    // TODO: Investigate why schema validation fails otherwise (see https://github.com/ChainSafe/lodestar/issues/6187)
-    routesDropOneOf: [
-      "produceBlockV2",
-      "produceBlockV3",
-      "produceBlindedBlock",
-      "publishBlindedBlock",
-      "publishBlindedBlockV2",
-    ],
+    routesDropOneOf: [],
```

Comment:
we should get rid of this option altogether, if we properly review the beacon api spec PRs, this issue should not happen in the future